sketch_map_tool/analyses/pdf_gen/pdf_gen.py

```diff
@@ -77,28 +81,41 @@ def add_result_page(result: AnalysisResult,
     canv.line(0, HORIZONTAL_LINE_Y, 1000, HORIZONTAL_LINE_Y)
 
     # Here we have one suggestion per analyses, not all recommendations together:
-    text = "<b>Result:</b><br/>"+result.message.replace("recommendations", "the suggestion")
+    text = "<b>Result:</b><br/>" + result.message.replace(
+        "recommendations", "the suggestion"
+    )
 
     if result.suggestion != "":
-        text += "<br/><br/><b>Suggestion:</b><br/>"+result.suggestion
-    image = RESOURCE_FOLDER+f"light_{result.level.value}.jpg"
-    canv.drawImage(image, CONTENTS_X, TRAFFIC_LIGHT_Y, TRAFFIC_LIGHT_WIDTH, TRAFFIC_LIGHT_HEIGHT,
-                   showBoundary=False)
+        text += "<br/><br/><b>Suggestion:</b><br/>" + result.suggestion
+    image = RESOURCE_FOLDER + f"light_{result.level.value}.jpg"
+    canv.drawImage(
+        image,
+        CONTENTS_X,
+        TRAFFIC_LIGHT_Y,
+        TRAFFIC_LIGHT_WIDTH,
+        TRAFFIC_LIGHT_HEIGHT,
+        showBoundary=False,
+    )
     canv.setFont(TEXT_FONT, TEXT_FONT_SIZE)
     style = ParagraphStyle(name="_", fontSize=TEXT_FONT_SIZE, fontName=TEXT_FONT)
     text_par = Paragraph(text, style=style)
-    w, h = text_par.wrapOn(canv, RESULT_TEXT_WIDTH, RESULT_TEXT_HEIGHT)  # pylint: disable=unused-variable,line-too-long  # noqa
-    text_par.drawOn(canv, RESULT_TEXT_X, RESULT_TEXT_Y-h)
+    _, h = text_par.wrapOn(
+        canv, RESULT_TEXT_WIDTH, RESULT_TEXT_HEIGHT
+    )  # pylint: disable=unused-variable,line-too-long  # noqa
```

Comment:
`line-too-long` can be removed here as it was only necessary to include the other pylint annotation on the same line.

Reply:
Removed the whole comment. It is unnecessary.